pkgs/applications/misc/rlaunch/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkgs
```

Comment:
please inherit them from `xorg` inside `all-packages.nix`
```suggestion
, libX11
, libXft
, libXinerama
```

Reply:
That is optional but pkgs needs to be dropped.